packages/ocaml-base-compiler/ocaml-base-compiler.4.09.1/opam

```diff
@@ -16,6 +16,7 @@ conflict-class: "ocaml-core-compiler"
 flags: compiler
 setenv: CAML_LD_LIBRARY_PATH = "%{lib}%/stublibs"
 build: [
+  ["sed" "-ib" "s/+dev1-2020-03-13//" "configure"]
```

Comment:
Is the use of sed fine to build on windows?

Reply:
`ocaml-base-compiler` doesn't presently work at all on Windows (that's what I'm working on at the moment!), so it's mildly moot 🙂

However, for the foreseeable future, Windows opam needs standard Unix tools available - it's got to run the `configure` script apart from anything else!

Reply:
That said, if you'd prefer I could do this as a patch file?

Reply:
Oh, if that is an option, I think it would be nicer